src/utils/item.js

```diff
@@ -0,0 +1,69 @@
+// eslint-disable-next-line no-useless-escape
```

Comment:
Do you think it's worth it to factor this whole file out? I think that it's used almost identically in all of our front ends.